core/src/main/java/io/grpc/transport/AbstractClientStream.java

```diff
@@ -125,8 +130,8 @@ protected void inboundHeadersReceived(Metadata.Headers headers) {
    *
    * @param frame the received data frame. Its ownership is transferred to this method.
    */
-  protected void inboundDataReceived(ReadableBuffer frame) {
-    Preconditions.checkNotNull(frame, "frame");
```

Comment:
Added a more readable message.  In the future, if finding out the version the library being used is important, it should be outputted somewhere, rather than developers having to derive it from a stack trace message.  As it stands, the existing single variable name error messages are basically useless to any library users (as opposed to developers, who likely know exactly what version they are running anyways)

I am worried about line wrapping, after coming from a significantly larger project than this one, and having to line wrap a ton.